setup.py

```diff
@@ -28,6 +28,7 @@
     ],
     setup_requires=[
         "nose>=1.0",
+        "mock>=1.0.1",
```

Comment:
Is mock required here? Seems like it should be sufficient to add to tests_require.